docs/mds/HIGH_FIDELITY_PROTOTYPE.md

```diff
@@ -0,0 +1,79 @@
+# Protótipo de Alta Fidelidade
+
+## Introdução
+Este documento tem como propósito apresentar o protótipo de alta fidelidade de forma que a parte visual seja o mais próximo possível do produto final.
+
+### Homepage
+![Homepage](https://i.imgur.com/QqMRpy2.png)
+
+A home é a página inicial do projeto. A partir dela o usuário pode explorar as funcionalidades do Dr. Down.
+
+## Sobre
+![Sobre](https://i.imgur.com/LwB7N5I.png)
+
+Nessa página ficam as informações sobre o CRIS DOWN.
+
+## Síndrome de Down
+![SD](https://i.imgur.com/MMDtf5w.png)
+
+Aqui ficam algumas informações sobre a doença e um pequeno FAQ para retirar dúvidas frequentes.
+
+## Registrar-se
+![signup](https://i.imgur.com/nDFoAhc.png)
+
+Página onde o usuário irá se cadastrar para poder utilizar as funcionalidades do projeto.
+
+## Registrar-se (Sucesso)
+![signupsucess](https://i.imgur.com/yLeAM6A.png)
+
+Mensagem de sucesso após se cadastrar corretamente.
+
+## Entrar
+![signin](https://i.imgur.com/52JTodb.png)
+
+Página onde o usuário, após ter se cadastrado, irá fazer login com seus dados.
+
+## Usuário logado
+![logado](https://i.imgur.com/tB1K4OE.png)
+
+Após o usuário ter feito login ele será redirecionado para a homepage. Clicando em "Meu perfil" um dropdown irá abrir com as seguintes opções: "Ver meu perfil, Consultas, Fila de espera, Fóruns e Eventos".
+
+## Meu perfil
+![perfil](https://i.imgur.com/Ze7WgcP.png)
+
+Página onde ficam as informações pessoais, acessíveis apenas para o próprio usuário logado. Aqui ele pode visualizá-las ou editá-las clicando em "Editar perfil".
```

Comment:
Tem a questão dos dados de cada tipo de usuário, quando for implementar tem que da atenção a isso

Reply:
Foi definido na reunião que não precisaria de todas as telas, só as mais importantes.